### PR TITLE
Style Autocomplete

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
### Description

Styling and stories for Autocomplete.

### CODE REVIEW

File: packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+  value: "The Godfather",
+};
+
+export const MaterialSingle = MuiTemplate.bind({});
+MaterialSingle.args = {};
+
+export const MaterialMultiple = MuiTemplate.bind({});
+MaterialMultiple.args = {
+  multiple: true,
+};
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx:276]: These are just for testing right?

File: packages/odyssey-react-mui/src/index.ts [hunk 2]
```diff
@@ -58,6 +59,7 @@ export {
   TableHead,
   TableRow,
   TableSortLabel,
+  TextField as TextFieldMui,
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/index.ts:62]: We'll wanna remove this in the final PR.

File: packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx (hunk 1 above)
Comment (edburyenegren-okta) [reply packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx:276]: Correct, just to check what is rendered when using MUI's TextField against ours.

File: packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.mdx [hunk 3]
```diff
@@ -68,10 +68,6 @@ Banner content should be succinct and direct. If possible, your content should b
 
 When including an action, be sure the link text clearly indicates where it leads.
 
-## Props
-
-<ArgsTable />
-
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.mdx:74]: Did you mean to remove this?

**Review by KevinGhadyani-Okta — APPROVED** [review]
Small comments, but **NON-BLOCKING**.

File: packages/odyssey-react-mui/src/theme/components.tsx [hunk 4]
```diff
@@ -539,10 +589,32 @@ export const components: ThemeOptions["components"] = {
             },
           },
         }),
+
+        [`.${inputBaseClasses.root}.${inputBaseClasses.disabled} &`]: {
+          backgroundColor: theme.palette.grey[200],
+        },
```
[the remark this answers is not in the document]
Comment (edburyenegren-okta) [reply packages/odyssey-react-mui/src/theme/components.tsx:595]: Yup. We don't have a token for it currently, but I'll be doing replacements with the new schema.

File: packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.mdx (hunk 3 above)
Comment (edburyenegren-okta) [reply packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.mdx:74]: Yup; in our current setup, these don't display anything. We had removed it from most files, but this one was lingering.

**Pull request merged**